Documentation/upgrade.md

```diff
@@ -0,0 +1,293 @@
+kind: Namespace
+metadata:
+  name: rook
+---
+apiVersion: rook.io/v1alpha1
+kind: Cluster
+metadata:
+  name: rook
+  namespace: rook
+spec:
+  versionTag: v0.5.0
+  dataDirHostPath: /var/lib/rook
+  storage:
+    useAllNodes: true
+    useAllDevices: true
+    storeConfig:
+      storeType: bluestore
+      databaseSizeMB: 1024
+      journalSizeMB: 1024
+```
+
+## Health Verification
+Before we begin the upgrade process, let's first review some ways that you can verify the health of your cluster, ensuring that the upgrade is going smoothly after each step.
+Most of the health verification checks for your cluster during the upgrade process can be performed with the Rook toolbox.
+For more information about how to run the toolbox, please visit the [Rook toolbox readme](./toolbox.md#running-the-toolbox-in-kubernetes).
+
+### Pods all Running
+In a healthy Rook cluster, the operator and all Rook namespace pods should be in the `Running` state and have zero pod restarts.
+To verify this, run the following commands:
+```bash
+kubectl get pods -l app=rook-operator -o wide
+kubectl -n rook get pod -o wide
+```
+If pods aren't running or are restarting due to crashes, you can get more information with `kubectl describe pod` and `kubectl logs` for the affected pods.
+
+### Status Output
+The Rook toolbox contains the `rookctl` command line tool that can give you status details of the cluster with the `rookctl status` command.
+Let's look at some sample output and review some of the details:
+```bash
+> kubectl -n rook exec -it rook-tools -- rookctl status
+OVERALL STATUS: OK
+
+USAGE:
+TOTAL       USED        DATA         AVAILABLE
+26.62 GiB   12.25 GiB   252.81 MiB   14.37 GiB
+
+MONITORS:
+NAME             ADDRESS             IN QUORUM   STATUS
+rook-ceph-mon0   10.3.0.48:6790/0    true        OK
+rook-ceph-mon1   10.3.0.140:6790/0   true        OK
+rook-ceph-mon2   10.3.0.197:6790/0   true        OK
+
+MGRs:
+NAME             STATUS
+rook-ceph-mgr0   Active
+
+OSDs:
+TOTAL     UP        IN        FULL      NEAR FULL
+6         6         6         false     false
+
+PLACEMENT GROUPS (100 total):
+STATE          COUNT
+active+clean   100
+```
+In the output above, note the following indications that the cluster is in a healthy state:
+* Overall status: The overall cluster status is `OK` and there are no warning or error status messages displayed.
+* Monitors:  All of the monitors are `in quorum` and have individual status of `OK`.
+* OSDs: All OSDs are `UP` and `IN`.
+* MGRs: All Ceph managers are in the `Active` state.
+* Placement groups: All PGs are in the `active+clean` state.
+
+If your `rookctl status` output has deviations from the general good health described above, there may be an issue that needs to be investigated further.
+
+### Pod Version
+The version of a specific pod in the Rook cluster can be verified in its pod spec output.  For example, for the monitor pod `mon0`, we can verify that it is running `v0.5.1` with the below commands:
```

Comment:
instead of saying "it is running v0.5.1", do you want to say "what version it is running"?

Reply:
changed the wording to remove the specific v0.5.1 reference